client/my-sites/domains/components/domain-warnings/pending-gapps-tos-notice-dialog.jsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+import PropTypes from 'prop-types';
+import React, { Fragment, useState } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import ClipboardButton from 'components/forms/clipboard-button';
+import { composeAnalytics, recordGoogleEvent, recordTracksEvent } from 'state/analytics/actions';
+import Dialog from 'components/dialog';
+import { getLoginUrlWithTOSRedirect } from 'lib/google-apps';
+import VerticalNav from 'components/vertical-nav';
+import VerticalNavItem from 'components/vertical-nav/item';
+import wp from 'lib/wp';
+
+function PendingGappsTosNoticeDialog( props ) {
+	const [ password, setPassword ] = useState( false );
+	const [ isCopied, setIsCopied ] = useState( false );
+	const translate = useTranslate();
+
+	const onPasswordClickHandler = e => {
+		e.preventDefault;
```

Comment:
We're missing parentheses here as well.

Reply:
Prettier removes parens on single param arrow functions.